plugins/kjob.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kjob
+spec:
+  version: v0.1.0
+  shortDescription: Tool for CLI-loving ML researchers.
```

Comment:
Please provide a useful description here. This sounds like a catchy billboard ad. :)

Reply:
+1

Reply:
Right, maybe something like this:
```
 Tool for CLI-loving ML researchers allowing them to run AI/ML jobs based on the pre-defined templates.
 ```

Reply:
Thanks! Done.